TestFoundation/TestNSString.swift

```diff
@@ -1228,7 +1240,7 @@ func checkHasPrefixHasSuffix(_ lhs: String, _ rhs: String, _ stack: [UInt]) -> I
 
 extension TestNSString {
     func test_PrefixSuffix() {
-#if !_runtime(_ObjC)
+//#if !_runtime(_ObjC)
```

Comment:
Through  this PR we are aligning the Linux behaviour with Darwin platform.Hence uncommenting  this as it is applicable to all platforms